app/javascript/packages/phone-input/index.js

```diff
@@ -42,15 +42,20 @@ export class PhoneInput extends HTMLElement {
   /** @type {string[]} */
   deliveryMethods = [];
 
+  /** @type {Object.<string,*>} */
+  countryCodePairs = {};
+
   connectedCallback() {
     /** @type {HTMLInputElement?} */
     this.textInput = this.querySelector('.phone-input__number');
     /** @type {HTMLSelectElement?} */
     this.codeInput = this.querySelector('.phone-input__international-code');
     this.codeWrapper = this.querySelector('.phone-input__international-code-wrapper');
     this.exampleText = this.querySelector('.phone-input__example');
+
     try {
       this.deliveryMethods = JSON.parse(this.dataset.deliveryMethods || '');
+      this.countryCodePairs = JSON.parse(this.dataset.translatedCountryCodeNames || '');
```

Comment:
Not that I would expect it to fail, but one downside of grouping these assignments in the same `try` block is that if `deliveryMethods` fails to parse, we won't assign `countryCodePairs`, even if `countryCodePairs` could be parsed.

To me, it seems like either we should trust that they'd always parse correctly and avoid the `try` blocks altogether, or if we want to be guarded about it, we should have two separate `try` blocks, one for each.

```suggestion
    } catch {}
    
    try {
      this.countryCodePairs = JSON.parse(this.dataset.translatedCountryCodeNames || '');
```